src/coreclr/nativeaot/System.Private.CoreLib/src/System/ValueType.cs

```diff
@@ -150,7 +137,9 @@ private unsafe int RegularGetValueTypeHashCode(ref byte data, int numFields)
                 }
                 else if (fieldType->IsPrimitive)
                 {
-                    hashCode = FastGetValueTypeHashCodeHelper(fieldType, ref fieldData);
+                    HashCode hash = default;
```

Comment:
The method should take `ref HashCode` argument instead of creating a new instance here.

Reply:
I meant change `RegularGetValueTypeHashCode` to take `ref HashCode` argument and  delete `HashCode hash = default;` that this comment is attached to.